diff --git a/node_modules/parse-entities/decode-entity.browser.js b/node_modules/parse-entities/decode-entity.browser.js
index feb3266..0983ce4 100644
--- a/node_modules/parse-entities/decode-entity.browser.js
+++ b/node_modules/parse-entities/decode-entity.browser.js
@@ -9,6 +9,18 @@ var semicolon = 59 //  ';'
 module.exports = decodeEntity
 
 function decodeEntity(characters) {
+  if (typeof document === 'undefined') {
+    var characterEntities = require('character-entities')
+    var own = {}.hasOwnProperty
+    return own.call(characterEntities, characters)
+      ? characterEntities[characters]
+      : false
+  } else {
+    return decodeEntityForBrowser(characters)
+  }
+}
+
+function decodeEntityForBrowser(characters) {
   var entity = '&' + characters + ';'
   var char
 
